server/database/models/User.js

```diff
@@ -11,15 +11,7 @@ const UserSchema = mongoose.Schema({
   email: {
     type: String,
     required: true,
-    validate: {
-      validator (email) {
-        User.find({ email }, (error, docs) => {
-          if (error) return console.error(error)
-          return docs.length === 0
-        })
-      },
-      message: 'User already exists!'
-    }
```

Comment:
A note on this: validate gets called anytime the user is saved (on creation or on update) so this wasn't working for our purposes, as an update request would fail with the message 'User already exists!'.

I believe `unique: true` will work here to disallow duplicate signups